pelican/plugins/test/data/pelicanconf.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- #
```

Comment:
The `coding` directive is not needed. As of [PEP 3120](https://www.python.org/dev/peps/pep-3120/), the default encoding for Python source is UTF-8. The shebang line can probably also be removed, since this file is not meant to be invoked directly.